components/suggestion/provider.tsx

```diff
@@ -10,21 +22,22 @@ export default class Provider {
         this.forceDispatch = false;
     }
 
-    handlePretextChanged(pretext, callback) { // eslint-disable-line no-unused-vars
+    // eslint-disable-next-line
```

Comment:
we are already disabling the linting in the next line ... so this is not necessary